amid/totalsegmentator/dataset.py

```diff
@@ -101,16 +96,19 @@ def affine(i, _base):
         """The 4x4 matrix that gives the image's spatial orientation"""
         file = f'{i}/ct.nii.gz'
 
-        with unpack(_base, file) as (unpacked, is_unpacked):
+        with unpack(_base, file, ARCHIVE_ROOT, '.zip') as (unpacked, is_unpacked):
             if is_unpacked:
                 return nibabel.load(unpacked).affine
             else:
                 with open_nii_gz_file(unpacked) as image:
                     return image.affine
 
+    def spacing(i, affine: Output):
+        return get_spacing_from_affine(affine)
```

Comment:
Maybe we should leave this to a transform? E.g. one that splits `affine` into `spacing`, `orientation` and `origin`

Reply:
got it, but don't know where I can place this transform

Reply:
`amid.nifty`?

Reply:
idk; maybe amid.transforms?

Reply:
yeah, ok